jobserver/templatetags/status_tools.py

```diff
@@ -0,0 +1,38 @@
+import structlog
+from django import template
+
+
+logger = structlog.get_logger()
+register = template.Library()
+
+
+#  A mapping of Job status codes -> descriptions of how to fix them
+status_hints = {
+    "tpp": {
+        "waiting_on_dependencies": "",
+        "dependency_failed": "",
+        "waiting_on_workers": "",
+        "nonzero_exit": "",
+    },
+}
+
+
+@register.simple_tag
+def status_hint(backend, status_code):
```

Comment:
I actually think this should take the entire job as an argument rather than just the `status_code` as there are things from the job (e..g the action name) we can use to help format a more useful message.